Frameworks/UIKit/UIPasteboard.mm

```diff
@@ -202,6 +202,97 @@ - (BOOL)containsPasteboardTypes:(NSArray*)pasteboardTypes {
 
 /**
  @Status Caveat
+ @Notes only support one item due to platform limit.
+ */
+- (NSArray<NSDictionary<NSString*, id>*>*)items {
+    WADDataPackageView* wasDataPackageView = _getClipboardContent();
+    id ret = [NSMutableDictionary dictionary];
+
+    if ([wasDataPackageView contains:[WADStandardDataFormats text]]) {
+        [ret setValue:[UIPasteboard _getStringFromDataPackageView:wasDataPackageView] forKey:(NSString*)kUTTypeText];
+    }
+
+    if ([wasDataPackageView contains:[WADStandardDataFormats bitmap]]) {
+        [ret setValue:[UIPasteboard _getImageFromDataPackageView:wasDataPackageView] forKey:(NSString*)kUTTypeImage];
+    }
+
+    if ([wasDataPackageView contains:[WADStandardDataFormats uri]]) {
+        [ret setValue:[UIPasteboard _getURLFromDataPackageView:wasDataPackageView] forKey:(NSString*)kUTTypeURL];
+    }
+
+    NSMutableArray<NSDictionary<NSString*, id>*>* items = [[[NSMutableArray alloc] init] autorelease];
+    if ([ret count] != 0) {
+        [items addObject:ret];
+    }
+
+    return items;
+}
+
+/**
+ @Status Caveat
+ @Notes only support one item
+ */
+- (void)setItems:(NSArray<NSDictionary<NSString*, id>*>*)items {
+    if (![_name isEqualToString:UIPasteboardNameGeneral]) {
+        TraceWarning(TAG, L"Only general pasteboard supported");
+        return;
+    }
+
+    if ([items count] > 0) {
+        if ([items count] > 1) {
+            TraceWarning(TAG, L"Only support first item in items, ignoring the rest");
+        }
+
+        WADDataPackage* dataPackage = [[WADDataPackage make] autorelease];
```

Comment:
>WADDataPackage* dataPackage = [[WADDataPackage make] autorelease]; [](start = 8, length = 66)

maybe use a StrongId to avoid the need for autorelease on this guy?